record.go

```
@@ -27,11 +28,14 @@ type Record struct {
 
 // newRecord returns a new instance of an existing record
 func newRecord(layout string, data interface{}, session Session) Record {
+	m := data.(map[string]interface{})
```

golangci-lint (2.11.4)

[error] type assertion must be checked

(forcetypeassert)